.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 **/.DS_Store
 **/**/.DS_Store
+.idea
```

Comment:
delete this

Reply:
ok removed; FYI this is to ignore intellij IDE project file